### ISSUE TITLE
Dropdown Documentation revision 

### ISSUE THREAD

**Issue by jelofson** [Dogfalo/materialize#5452]
https://github.com/Dogfalo/materialize/blob/fdca3eea1709f313fb2a23d7f82ad73a35d5b2ac/jade/page-contents/dropdown_content.html#L35

Dropdowns actually DO need to initialized manually. As per conversation with @DanielRuf on gitter.

**Comment by jelofson** [comment]
Simple example of initializing several dropdowns at once
```javascript
var dropdownElems = document.querySelectorAll('.dropdown-trigger');
var dropdowns = [];
for (var i=0; i<dropdownElems.length; i++) {
  dropdowns[i] = new M.Dropdown(dropdownElems[i]);
}
```

**Comment by acburst** [comment]
Fixed in d0b70afa
